browser.js

```diff
@@ -16,8 +16,8 @@ exports.pbkdf2 = function (password, salt, iterations, keylen, digest, callback)
 }
 
 exports.pbkdf2Sync = function (password, salt, iterations, keylen, digest) {
-  if (!Buffer.isBuffer(password)) password = new Buffer(password, 'binary')
-  if (!Buffer.isBuffer(salt)) salt = new Buffer(salt, 'binary')
+  if (!Buffer.isBuffer(password)) password = new Buffer(password, 'utf-8')
+  if (!Buffer.isBuffer(salt)) salt = new Buffer(salt, 'utf-8')
```

Comment:
You don't need second argument here now, encoding is 'utf8' by default.
Maybe even use `Buffer.from(..)` ?